sample-info/brain-classifications.tsv

```diff
@@ -0,0 +1,29 @@
+ontology_id	human_readable_value	submitted_diagnosis	diagnosis_group	subdiagnosis_group
+MONDO:0016684	Anaplastic astrocytoma	Anaplastic astrocytoma	Brain and CNS	High-grade glioma
+MONDO:0016700	Anaplastic ependymoma	Anaplastic ependymoma	Brain and CNS	NA
+MONDO:0016734	Anaplastic ganglioglioma	Anaplastic ganglioglioma	Brain and CNS	NA
+MONDO:0021640	Grade III glioma	Anaplastic glioma	Brain and CNS	High-grade glioma
+MONDO:0020560	Atypical teratoid rhabdoid tumor	Atypical teratoid rhabdoid tumor	Brain and CNS	High-grade glioma
+MONDO:0022965	Desmoplastic infantile ganglioglioma	Desmoplastic ganglioglioma	Brain and CNS	NA
+MONDO:0005499	Brain glioma	Diffuse midline glioma	Brain and CNS	High-grade glioma
+MONDO:0005505	Dysembryoplastic neuroepithelial tumor	Dysembryoplastic neuroepithelial tumor	Brain and CNS	Low-grade glioma

```

Comment:
If a ganglioglioma is not a low-grade glioma, neither is this. Both are glioneuronal and neuronal tumors: https://pmc.ncbi.nlm.nih.gov/articles/PMC8328013/#T1